doc/rpki.texi

```diff
@@ -14,15 +14,15 @@ cryptographic data (so called Route Origin Authorization, or short
 @acronym{ROA}, objects) will be performed by trusted cache servers. The
 RPKI/RTR protocol defines a standard mechanism to maintain the exchange of
 the prefix/origin AS mapping between the cache server and routers.
-In combination with a  BGP prefix origin validation scheme a router is able
+In combination with a  BGP Prefix Origin Validation scheme a router is able
 to verify received BGP updates without suffering from cryptographic
 complexity.
 
 
 The RPKI/RTR protocol is defined in @cite{RFC6810, The Resource Public Key
 Infrastructure (RPKI) to Router Protocol}, and the validation scheme in
 @cite{RFC6811, BGP Prefix Origin Validation}. The current version of Prefix
-Origin Validation in Quagga implements both RFCs.
+Origin Validation in FRRouting implements both RFCs.
```

Comment:
looking through other docs its mostly referred to as plain and short `FRR` not `FRRouting`